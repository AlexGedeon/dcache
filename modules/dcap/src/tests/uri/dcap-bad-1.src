dcap:/abs-path/to/file
