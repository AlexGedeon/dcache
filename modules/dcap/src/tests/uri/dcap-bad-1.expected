URL: dcap:/abs-path/to/file
  dc_getURL() returned NULL
